tools/nodejs/src/data_chunk.cpp

```diff
@@ -145,7 +145,19 @@ Napi::Array EncodeDataChunk(Napi::Env env, duckdb::DataChunk &chunk, bool with_t
 				}
 				break;
 			}
-			case duckdb::LogicalTypeId::BLOB:
+			case duckdb::LogicalTypeId::BLOB: {
+				if (with_data) {
```

Comment:
Completely unrelated to this PR - but `with_data` and `with_types` on this method appear to me like legacy parameters. `EncodeDataChunk` does not appear to be called anywhere where these settings are not both set to true. @Mause is that correct or am I missing something?